test/test_art.py

```diff
@@ -26,9 +26,19 @@
 from beets import library
 from beets import importer
 from beets import config
+from beets import logging
 
 
-class FetchImageTest(_common.TestCase):
+logger = logging.getLogger('beets.test_art')
+
+
+class UseThePlugin(_common.TestCase):
+    def setUp(self):
+        super(UseThePlugin, self).setUp()
+        self.plugin = fetchart.FetchArtPlugin()
```

Comment:
This (instantiating a plugin object for the scope of a test) seems like a useful pattern. Maybe we should do more of that.